modules/ServerAPI/src/server/routes/ExperimentServiceRoutes.coffee

```diff
@@ -140,11 +140,11 @@ exports.experimentByName = (req, resp) ->
 		serverUtilityFunctions.getFromACASServer(baseurl, resp)
 
 exports.experimentsByProtocolCodename = (req, resp) ->
```

Comment:
All this code does is look up the users allowed projects and pass them as a query param to the backend which will filter experiments based on acls